openjdk/mmtk.h

```diff
@@ -11,6 +11,10 @@ extern "C" {
 
 typedef void* MMTk_Mutator;
 typedef void* MMTk_TraceLocal;
+typedef enum {
```

Comment:
Possibly add comments to state that this has the same layout as `mmtk::util::alloc::MmtkAllocationError`.